src/Pim/Bundle/EnrichBundle/Normalizer/ProductNormalizer.php

```diff
@@ -158,7 +159,7 @@ public function normalize($product, $format = null, array $context = [])
             'updated'           => $updated,
             'model_type'        => 'product',
             'structure_version' => $this->structureVersionProvider->getStructureVersion(),
-            'completenesses'    => $this->getNormalizedCompletenesses($product)
+//            'completenesses'    => $this->getNormalizedCompletenesses($product)
```

Comment:
Same than the updater, I think it is not a good idea to a have a generic normalizer